lib/msal-common/src/utils/Constants.ts

```diff
@@ -4,7 +4,8 @@
  */
 
 export const Constants = {
-    LIBRARY_NAME: "MSAL.JS",
+    LIBRARY_NAME: "MSAL.JS", // TODO JS should be broken up into common, browser, node. What dore core/angular use?
+    SKU: "MSAL.JS.common",
```

Comment:
What should this SKU be? SKU should be different for msal-node, msal-browser, and msal-common. 

For msal-node it's "msal.node"

Reply:
msal.common should not have a SKU IMO, only implementations (node and browser) should update the SKU.

Reply:
> msal.common should not have a SKU IMO, only implementations (node and browser) should update the SKU.

Agreed.

Reply:
@sameerag @jasonnutter why wouldn't msal-common have a SKU? We're making msal-common available to be consumed directly, why wouldn't we want to be able to track who is using it directly? 

When we are running telemetry queries, I think it would be valuable to see how many people are using msal-node, msal-browswer, or just using msal-common directly. SKU has no other effect than letting us split this data out, not sure why we wouldn't just add it.

Reply:
I think if you are using common, you should have to pass in a SKU to identify yourself uniquely.

Reply:
The problem is that I'm pretty sure 95%+ of people wouldn't pass that value in, and so we would never get that data. I think when these headers were originally added they were meant to be used internally only anyways.  

We also have applicationName and applicationVersion in TelemetryOptions which serves the purpose letting people pass in this info if so they desire. IMO MSAL SKU shouldn't be optional.

Reply:
I agree that SKU should be required when using common.

Reply:
Well, okay. Only concern I have is will we able to trace all `msal-common` data to `msal-browser` vs `msal-ndode`? All service requests originate from `msal-common`, so are we overwriting the SKU in that case? Will this be triggered only if someone manages to write their wrapper and calls into `msal-common`?

Reply:
Requiring any library that is using `msal-common` to provide a SKU would specifically allow us to be able to differentiate between different libraries, if I'm understanding your concern. If requests have `msal-common` as the SKU, we may not be able to directly know which library actually generated the request.

Reply:
I agree `msal-common` doesn't need it's own SKU, and we can make it a required config param for `msal-common`. We should be careful with this value because the service uses it to make decisions on which errors to throw back. We need to check with the service to see how the behavior will differ based on which SKU value is given back.

Reply:
Discussed in design meeting, agreed that it makes sense to have SKU for msal-common, and default to this SKU if none is passed in. Resolving.